Drivers/seds/AfecHwas.xml

```diff
@@ -0,0 +1,84 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Why is this modified in the scope of this MR?

Reply:
Besides implementing lidar, I have also split every SEDS file into interface and implementation. I can try to isolate this change into a separate branch if you like.

Reply:
OK, so this is just this split? I will review it in more detail then.